monitor/redis/watcher.go

```diff
@@ -104,14 +102,17 @@ func (w *Watcher) getValues(ctx context.Context, ch chan<- []*change.Change) {
 		w.versions[i]++
 		w.hashes[i] = hash
 
-		changes = append(changes, change.New(config.SourceRedis, key, value, w.versions[i]))
+		cg := change.New(config.SourceRedis, key, value, w.versions[i])
+		if cg != nil {
```

Comment:
I hesitated to also change the `change.New` method to return a `Change` instead of `*Change`: I would be favourable to this change but I think we already have enough to discuss in the scope of this pr.